blocks/form/form.js

```diff
@@ -348,6 +348,38 @@ function inputDecorator(field, element) {
   }
 }
 
+function decoratePanelContainer(panelDefinition, panelContainer) {
+  if (!panelContainer) return;
+
+  const isPanelWrapper = (container) => container.classList?.contains('panel-wrapper');
```

Comment:
you should check if these conditions are required in other methods as well in future. In that case rather than having them as method scope variable, you should expose them as private static methods to reused in future as well

Reply:
Rest looks good to me

Reply:
I did check, they were not required elsewhere and so I kept it in the same scope